### PR TITLE
Travis CI support

### PR DESCRIPTION
**Pull request by Haegin**
Adds support to Travis CI. When we merge this master will build and we'll get the badge on the README updating.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
 language: ruby
+bundler_args: --without debugging local_testing
 rvm:
-  - 2.1.2
+  - 1.9.3
```
Comment (caius) [inline .travis.yml]: Do we need to support 1.9 anymore? It's dead.

Comment (Haegin) [inline .travis.yml]: The gem says it supports 1.9.3 at the moment.

--
Regards,
Harry Mills

Comment (caius) [inline .travis.yml]: But we're going to kill that, so why bother building tests for it :grin:

**Comment by caius** [comment]
:+1: Pass

**Pull request merged**